stubs/Authlib/authlib/integrations/base_client/async_app.pyi

```diff
@@ -1,18 +1,20 @@
 from logging import Logger
+from typing import Any
 
+import httpx
 from authlib.integrations.base_client.sync_app import OAuth1Base, OAuth2Base
 
 log: Logger
 
 __all__ = ["AsyncOAuth1Mixin", "AsyncOAuth2Mixin"]
 
 class AsyncOAuth1Mixin(OAuth1Base):
-    async def request(self, method, url, token=None, **kwargs): ...
-    async def create_authorization_url(self, redirect_uri=None, **kwargs): ...
-    async def fetch_access_token(self, request_token=None, **kwargs): ...
+    async def request(self, method: str, url: str, token: dict[str, Any] | None = None, **kwargs) -> httpx.Response: ...
+    async def create_authorization_url(self, redirect_uri: str | None = None, **kwargs) -> dict[str, Any]: ...
+    async def fetch_access_token(self, request_token: dict[str, Any] | None = None, **kwargs) -> dict[str, Any]: ...
 
 class AsyncOAuth2Mixin(OAuth2Base):
-    async def load_server_metadata(self): ...
-    async def request(self, method, url, token=None, **kwargs): ...
-    async def create_authorization_url(self, redirect_uri=None, **kwargs): ...
-    async def fetch_access_token(self, redirect_uri=None, **kwargs): ...
+    async def load_server_metadata(self) -> dict[str, Any]: ...
+    async def request(self, method: str, url: str, token: dict[str, Any] | None = None, **kwargs) -> httpx.Response: ...
```

Comment:
Don't forget to type `**kwargs` if your touching this function. The same applies for all others.